app/components/provider_interface/application_rejection_feedback_component.rb

```diff
@@ -0,0 +1,49 @@
+module ProviderInterface
+  class ApplicationRejectionFeedbackComponent < ViewComponent::Base
+    attr_reader :application_choice
+
+    def initialize(application_choice:)
+      @application_choice = application_choice
+    end
+
+    def render?
+      application_choice.rejected?
```

Comment:
I don't think it will be an issue because we have the redirect in place and a condition on displaying the 'Feedback' tab but an application rejected by default may not have any feedback for a while.

So something like the below would be more accurate.
```suggestion
      application_choice.rejected? && (application_choice.rejection_reason.present? || application_choice.structured_rejection_reasons.present?)
```
or `application_choice.rejected? && !application_choice.no_feedback?`